tests/test_integration_scenarios.py

```
@@ -177,27 +181,42 @@ def test_complete_meshtastic_to_matrix_flow(self):
             "mmrelay.matrix_utils.matrix_relay", new_callable=AsyncMock
         ) as mock_matrix_relay:
-            with patch("mmrelay.plugin_loader.load_plugins") as mock_load_plugins:
-                # Mock debug plugin
-                mock_plugin = MagicMock()
-                mock_plugin.handle_meshtastic_message = AsyncMock(return_value=False)
-                mock_load_plugins.return_value = [mock_plugin]
+            with patch("mmrelay.meshtastic_utils._submit_coro") as mock_submit_coro:
+                import inspect
+                from concurrent.futures import Future
+
+                def _done_future(coro, *args, **kwargs):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_done_future`

(ANN202)